packages/itwin/reports-config-widget/src/widget/components/ReportMappings.tsx

```diff
@@ -59,9 +59,7 @@ enum ReportMappingsView {
 }
 
 const fetchReportMappings = async (
-  setReportMappings: React.Dispatch<
-  React.SetStateAction<ReportMappingAndMapping[]>
-  >,
+  setReportMappings: React.Dispatch<React.SetStateAction<ReportMappingAndMapping[]>>,
```

Comment:
There is no point of having such a long type. Change to `(x: ReportMappingAndMapping[]) => void`

Reply:
Type has been simplified